src/components/CalendarMonth.jsx

```diff
@@ -45,6 +45,7 @@ const propTypes = forbidExtraProps({
   renderDayContents: PropTypes.func,
   firstDayOfWeek: DayOfWeekShape,
   setMonthHeight: PropTypes.func,
+  verticalSpacing: nonNegativeInteger,
```

Comment:
I think this name is somewhat overloaded as `verticalSpacing` in the case of the SDP and DRP signals the margin between the calendar dropdown and the inputs. Could we name this either `verticalBorderSpacing` to allude to the table or `verticalWeekSpacing` to allude to the calendar contents?